config.go

```diff
@@ -614,6 +615,19 @@ func loadConfigFile(preCfg *Config, interceptor signal.Interceptor) (*Config,
 	// configuration is fully valid. This also sets up the main logger that
 	// logs to a sub-directory in the .lnd folder.
 	case ModeIntegrated:
+		// For the integration of tapd with lnd, we need to allow tapd
+		// to send custom error messages to peers through the
+		// SendCustomMessage RPC in lnd. Since the error messages aren't
+		// in the custom range, we explicitly need to allow them. This
+		// isn't currently needed in remote mode, because custom
+		// channels are only available if both lnd and tapd are running
+		// in integrated mode. We need to set this value before we call
+		// lnd.ValidateConfig() below, because that's what's going to
+		// inject these values into the lnwire package.
+		cfg.Lnd.ProtocolOptions.CustomMessage = append(
+			cfg.Lnd.ProtocolOptions.CustomMessage, lnwire.MsgError,
```

Comment:
if tapd is disabled, maybe we should not force this? 

(note to self that maybe we can eventually hide this type of thing behind the `SubServer` interface so that we can be sure that any config tweaks only happen if a sub-server is active. But yeah - i'll take care of that refactor)

Reply:
Hmm, at this point in parsing the config we haven't really done any validation for the taproot asset mode yet, so not sure it's wise to access that here?
Also, this isn't really super consequential, it just allows an RPC user to send p2p error messages to the peers. So no big deal if this is enabled IMO.